packages/forms/src/GroupControl/GroupControl.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { useField } from 'react-final-form'
+
+import { Field } from '@dhis2/ui-core'
+import propTypes from '@dhis2/prop-types'
+
+export const GroupControl = ({ name, label, children, required }) => {
+    const {
+        meta: { error, touched },
+    } = useField(name, { subscription: { error: true, touched: true } })
+
+    const isError = !!error && !!touched
+    let errorText
+
+    if (isError) {
+        if (typeof error === 'string') {
+            errorText = error
+        } else if (typeof error === 'object' && error[name]) {
+            errorText = error[name]
+        } else {
+            errorText = null
+        }
+    }
+
+    return (
+        <Field
```

Comment:
Should this not be a FieldSetField, if it's expected to contain a group of form inputs?

Reply:
Yeah: efa7b83